service/src/test/java/uk/nhs/adaptors/gp2gp/uat/EhrExtractUATTest.java

```diff
@@ -163,6 +166,7 @@ public void tearDown() {
         messageContext.resetMessageContext();
     }
 
+    @Disabled
```

Comment:
If this is long-term, can you add a description of why it's disabled?
```suggestion
    @Disabled("Cuz it works worse'n a chocolate teapot")
```
otherwise fix!
```suggestion
```

Reply:
Actually, even if it's temporary within this piece of work, might still be a good idea to add a value to the annotation stating as such